src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -375,13 +419,27 @@ def get_suggestion(self, random_search=False):
         total_params : dict
             parameter suggestion
         """
+        if self.parallel and len(self.total_data)>20 and len(self.running_data):
+            self.CL_rval = copy.deepcopy(self.rval)
+            _constant_liar_y = 0
+            if self.constant_liar_type == 'mean' and self.optimal_y[1]:
+                    _constant_liar_y = self.optimal_y[0] / self.optimal_y[1]
+            else:
+                _constant_liar_y = self.optimal_y
+            for _parameter_id in self.running_data:
+                self.receive_trial_result(parameter_id=_parameter_id, parameters=None, value=_constant_liar_y, constant_liar=True)
+            rval = self.CL_rval
 
-        rval = self.rval
+            random_state = np.random.randint(2**31 - 1)
```

Comment:
Duplicate code, can move out of "if-else" circle? and curious of why set random_state to a random value?

Reply:
because there is a bug in the previous version. If you reuse the CL_rval, then the seed will be same, so the previous version always generate the same config in 0-20th config.

Reply:
I see. Good point! thx~